examples/kubernetes/daemon-set-log-collection/daemon-set-log-collection.yml

```diff
@@ -30,6 +30,7 @@ data:
     nifi.provenance.repository.max.storage.time=1 MIN
     nifi.provenance.repository.max.storage.size=1 MB
     nifi.flowfile.repository.directory.default=${MINIFI_HOME}/flowfile_repository
+    nifi.flowfile.checkpoint.directory.default=${MINIFI_HOME}/flowfile_checkpoint
```

Comment:
I think the property is not relevant here, so I would leave it out from the example. I would ideally also remove all other properties that are using their default values, to keep the example minimal, but that doesn't need to happen in this PR.

Reply:
Added jira ticket for this impovement: https://issues.apache.org/jira/browse/MINIFICPP-2004